scripts/build-fontawesome.js

```diff
@@ -3,11 +3,8 @@
 var fs = require("fs-extra");
 
 var srcDir = "./node_modules/font-awesome/fonts/";
-var destDir = "./client/fonts/";
+var destDir = "./client/css/fonts/";
```

Comment:
@xPaw, this was pulled to `client/fonts/` on purpose. I believe all fonts should be there instead of `client/css/fonts/`, but this is also a way to locate the fonts installed by our (up and coming) asset pipeline differently.
What do you think?